docs/reference/search/request/highlighting.asciidoc

```diff
@@ -45,35 +43,43 @@ from versions before 5.0) that match the expression to be highlighted.
 Note that all other fields will not be highlighted. If you use a custom mapper and want to
 highlight on a field anyway, you have to provide the field name explicitly.
 
-[[plain-highlighter]]
-==== Plain highlighter
+[[unified-highlighter]]
+==== Unified Highlighter
 
-The default choice of highlighter is of type `plain` and uses the Lucene highlighter.
-It tries hard to reflect the query matching logic in terms of understanding word importance and any word positioning criteria in phrase queries.
+The default choice of highlighter is of type `unified` and uses the Lucene Unified highlighter.
```

Comment:
The `unified` highlighter (which is used by default if no highlighter `type` is specified) uses the Lucene Unified Highlighter.